idr/.snapshots/TestJSONify2XML

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
is this a special json file, why the key are like this "xmlns:a", "#attributes"

Reply:
All files in `.snapshots` folders are, well, snapshot files from tests.

JSON does allow these chars in property names. And you can access them in JavaScript like this:

`var n = JSON.parse(...); console.log(n["#attributes]);`